welcome-to-sourcery.py

```diff
@@ -0,0 +1,34 @@
+
+
+# Sourcery Installation
+
+# 1. Go to https://sourcery.ai/download/?editor=vscode to get a free token.
+# 2. Enter the token into the box above, or search for `sourcery` in the
+#    settings and enter the token into the `Sourcery Token` field.
+
+
+# Welcome to Sourcery! Once you've added the token, look for the
+# underlined `result = []` below. This is a suggestion from Sourcery.
+# Hover over it to see details of the changes including a diff. 
+
+def refactoring_example(spellbook):
+    result = []
+    for spell in spellbook:
+        if spell.is_awesome:
+          result.append(spell)
+    return result
+
+# Apply the suggestion by taking the quick-fix action:
+#   Put the cursor on the line and either click the lightbulb or use the hotkey:
+#   (Ctrl + .) or (Cmd + .) by default. Take the 'Convert for loop...' option.
```

Comment:
The style of these keyboard shortcuts is different from the line before.

Reply:
reworded